WorkingWithText/Program.cs

```diff
@@ -39,15 +131,53 @@ public static bool IsValidTime(string hyphenNum)
         // Trim off unneeded spaces.
         public static string PascalConverter(string aFewWords)
         {
-        	return default;
+            // First check if there's something to work off of, if not we're done!
+            if (string.IsNullOrWhiteSpace(aFewWords))
+            {
+                return null;
+            }
+
+            // then we make some variables, making sure to trim any spaces off out input
+            // to be sure they don't get in the way of our splitting
+            var myFewWords = aFewWords.Trim(' ');
+            var inputPascal = string.Empty;
+
+            // split the string into an array based off the spaces beteen words
+            foreach (var word in myFewWords.Split(' '))
+            {
+                // capitalize the first char of each word and lower the rest
+                var capitalWord = char.ToUpper(word[0]) + word.ToLower().Substring(1);
+                // and put them back together! and add them to our empty string
+                inputPascal += capitalWord;
+            }
+            return inputPascal;
         }
 
         // 5- Write a method that accepts an English word. Count the number of vowels
         // (a, e, i, o, u) in the word. So, if the user enters "inadequate", the program should
         // return 6.
         public static int VowelCounter(string aWord)
         {
-            return default;
+            // make sure our input is all in lowercase format
+            var wordArr = aWord.ToLower().ToCharArray();
+            int result = 0;
+
+            // check to see if any char in our array matches a vowel
+            foreach (var letter in wordArr)
+            {
+                if (letter == 'a')
+                    // if they do then we increase our result by 1
+                    result++;
+                else if (letter == 'e')
+                    result++;
+                else if (letter == 'i')
+                    result++;
+                else if (letter == 'o')
+                    result++;
+                else if (letter == 'u')
+                    result++;
+            }
+            return result;
```

Comment:
I like the foreach loop, however the if statements could be condensed into one to reduce complexity. For ex:
```
if (letter == 'a' || letter == 'e' || letter == 'i' || letter == 'o' || letter == 'u')
{
    result++;
}
```
Using the above line checks if the char is a vowel and increments result without checking it against each vowel individually.